webpack/assets/javascripts/react_app/components/common/table/actions/getTableItemsAction.test.js

```diff
@@ -4,7 +4,7 @@ import { ajaxRequestAction } from '../../../../redux/actions/common';
 jest.mock('../../../../redux/actions/common');
 describe('getTableItemsAction', () => {
   it('should call ajaxRequestAction with url ', () => {
-    const controller = 'models';
+    const tableID = 'models';
```

Comment:
can the id be more specific ? in the state this is stored as `models` with no indication that it represents a table
![Screenshot from 2019-07-09 16-39-15](https://user-images.githubusercontent.com/11807069/60892795-6a95a000-a268-11e9-83f9-a7d0b32de1a7.png)

Reply:
Added `MODELS_TABLE_ID = 'models_table';` const to solve this issue